app/domain/authentication/authn_jwt/authentication_parameters.rb

```diff
@@ -0,0 +1,24 @@
+module Authentication
+  module AuthnJwt
+    # Data class to store data regarding jwt token that is needed during the jwt authentication process
+    class AuthenticationParameters
```

Comment:
This data class looks very specific to `jwt`.
Please keep in mind there are other authenticators.
would you expect them to create such class each?
Maybe you could come with a better approach to unify it?